alpine/Dockerfile

```diff
@@ -16,7 +16,10 @@ RUN adduser -Su 1337 kong \
 	&& rm -rf /tmp/usr \
 	&& cp -R /tmp/etc / \
 	&& rm -rf /tmp/etc \
-	&& apk del .build-deps
+	&& apk del .build-deps \
+	&& chown -R kong /usr/local/kong \
+	&& chgrp -R 0 /usr/local/kong \
```

Comment:
No need for a separate `chgrp`, above line can be `chown -R kong:0 /usr/local/kong`.